agenta-backend/agenta_backend/routers/evaluation_router.py

```diff
@@ -491,6 +492,34 @@ async def create_custom_evaluation(
         status_code=200,
     )
 
+@router.put("/edit_custom_evaluation/{id}")
```

Comment:
let's use rest api best practices.
let's make it /evaluations/custom_evaluations/{id}